core/editor/src/consts.rs

```diff
@@ -11,4 +11,6 @@ pub const MOUSE_ZOOM_RATE: f64 = 1. / 400.;
 
 pub const ROTATE_SNAP_INTERVAL: f64 = 15.;
 
-pub const SELECTION_TOLERANCE: f64 = 5.0;
+pub const LINE_ROTATE_SNAP_ANGLE: f64 = 12.;
```

Comment:
Wondering if this should be 15. However this already was 12 beforehand, yet the actual application does work in 15° increments while holding <kbd>Shift</kbd>.